apsi-enrollment-be/src/main/java/edu/pw/apsienrollment/qrcode/exception/QRCodeGenException.java

```diff
@@ -0,0 +1,10 @@
+package edu.pw.apsienrollment.qrcode.exception;
+
+import edu.pw.apsienrollment.common.exception.ApsiException;
+import edu.pw.apsienrollment.common.exception.ExceptionCode;
+
+public class QRCodeGenException extends ApsiException {
+    public QRCodeGenException() {
+        super(ExceptionCode.QR_CODE_GEN_FAILED, "Error during QR code generation", null);
```

Comment:
Please, add translation of this error code on FE

Reply:
Done